src/main/java/io/jenkins/plugins/orka/OrkaCloud.java

```diff
@@ -162,10 +162,12 @@ public Node call() throws Exception {
 
                 OrkaProvisionedAgent agent = template.provision();
 
-                logger.fine(provisionIdString + "Adding Node to Jenkins:");
-                logger.fine(agent.toString());
+                if (agent != null) {
```

Comment:
Maybe we can log a warning message when we're not able to create an agent.

Reply:
It is logged [here](https://github.com/jenkinsci/macstadium-orka-plugin/pull/25/files#diff-4fbb7924f2c1ec6563ef7377c222f4d3R152)